app/assets/v2/js/cart.js

```diff
@@ -220,36 +219,7 @@ Vue.component('grants-cart', {
 
       return result;
     },
-    filterByNetworkId: function() {
-      const vm = this;
-      let result;
-      let networkId;
-
-      networkId = vm.tokenListOptions.networkId || vm.networkId;
-
-      if (vm.networkId == '') {
-        result = vm.filterByChainId;
-      } else if (vm.tokenListOptions.strict) {
-        result = vm.sortByPriority.filter((item) => {
-          return String(item.networkId) == networkId;
-        });
-        return result;
```

Comment:
removing this will break algorand cross-chain token filtering https://github.com/gitcoinco/web/pull/10260

Reply:
`filterByNetworkId` is currently unused - are you sure?

Reply:
observe `vm.tokenListOptions.strict`

it's used in `tabChange` method L645 and L684-L686